Stratis.Bitcoin.Features.Wallet.Tests/WalletSyncManagerTest.cs

```diff
@@ -89,6 +89,9 @@ public void Start_BlockNotChain_ReorgsWalletManagerUsingWallet()
             Assert.Equal(walletSyncManager.WalletTip.HashBlock.ToString(), forkBlock.HashBlock.ToString());
         }
 
+        /// <summary>
+        /// When processing a new block that has a previous hash that is the same as the wallettip pass it directly to the walletmanager and set it as the new tip.
```

Comment:
Consider using `<see cref="xyz">` when you talk about WalletTip and WalletManager. 
Also below and also for BlockStoreCache (also not sure why you have lowercase everywhere) and WalletSyncManager as well.

Reply:
I will update it.